schema.sql

```diff
@@ -1,10 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-  id serial,
+  id INT,
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 It would also be great to make the `id` column not null and autoincremented and also the primary key 💯